xbmc/peripherals/devices/Peripheral.h

```diff
@@ -8,7 +8,6 @@
 
 #pragma once
 
-#include "XBDateTime.h"
```

Comment:
Isn't this needed because CDateTime is returned by value below? If not, why is it included in the peripheral subclass headers?

Reply:
Return by value is fine. However, in the Peripheral classes there are members of type CDateTime.